geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -252,4 +277,28 @@ public String toString() {
   public KnownVersion[] getSerializationVersions() {
     return null;
   }
+
+  protected byte[] processIncrement(String stringIncr) {
+    double incr;
+    switch (stringIncr) {
+      case "inf":
+      case "+inf":
+      case "infinity":
+      case "+infinity":
+        incr = POSITIVE_INFINITY;
+        break;
+      case "-inf":
+      case "-infinity":
+        incr = NEGATIVE_INFINITY;
+        break;
+      default:
+        try {
+          incr = Double.parseDouble(stringIncr);
+        } catch (NumberFormatException nfe) {
+          return ERROR_NOT_A_VALID_FLOAT.getBytes();
```

Comment:
At this point, the error should just be left to bubble up and be processed 'normally'. You can just do:
```
throw new NumberFormatException(ERROR_NOT_A_VALID_FLOAT);
```